src/index.js

```diff
@@ -170,35 +171,38 @@ export default class ReactMarkdownGithub extends Component {
   * @api private
   */
   render() {
-    const { className, source, resolver } = this.props;
     const renderers = {
       heading: this.renderHeading,
       ...this.props.renderers
     };
 
+    const cleanedProps = rip(
+      this.props,
+      'renderers',
+      'transformImageUri',
+      'transformLinkUri'
+    );
```

Comment:
If you do the spread first in JSX this shouldn't be necessary, right? e.g.

``` js
render() {
  const renderers = {
    heading: this.renderHeading,
    ...this.props.renderers
  };	     

  return <ReactMarkdown { ...this.props }
    renderers={ renderers }
    transformLinkUri={ this.transformLinkUri }
    transformImageUri={ this.transformImageUri } />;
}
```

Reply:
Actually, that *would* work now. Originally, I suggested that to be able to override the `transformLinkUri` and `transformImageUri` calls, but that’s no longer needed since they’re called as callbacks in those functions. Let’s do it that way, *as long as* we don’t change it in the future to not allow the user to provide their own override of a prop.

Reply:
Agreed. @QuicheSama please update this to use the JSX spread technique outlined in my previous comment.

Reply:
Addressed in https://github.com/godaddy/react-markdown-github/pull/5/commits/46b015826d8f9cd8b6d6ce61d5f4f9782082203f